.changeset/six-eggs-eat.md

```diff
@@ -0,0 +1,7 @@
+---
+'@sap-ux-private/preview-middleware-client': patch
+'@sap-ux/adp-tooling': patch
+'@sap-ux/preview-middleware': patch
+---
```

Comment:
Please add patch to "preview-middleware" as well

Reply:
But it is there? :)